openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -64,32 +64,37 @@ <h2 class="hd hd-3 page-title">${course.display_name_with_default}</h2>
                     </div>
                 % endif
 
-                ${HTML(outline_fragment.body_html())}
+                % if outline_fragment:
+                    ${HTML(outline_fragment.body_html())}
+                % endif
             </main>
             <aside class="course-sidebar layout-col layout-col-a">
                 <%
-                course_tools = CourseToolsPluginManager.get_course_tools()
+                course_tools = filter(
```

Comment:
Nit/Question: Do we have any practice around trying to minimize code in template, and could/should we compute outside and pass this in?